index.js

```diff
@@ -17,7 +17,12 @@ var WinstonCloudWatch = function(options) {
   var awsSecretKey = options.awsSecretKey;
   var awsRegion = options.awsRegion;
   var messageFormatter = options.messageFormatter ? options.messageFormatter : function(log) {
-    return [ log.level, log.msg, stringify(log.meta) ].join(' - ');
+    if(stringify(log.meta)==="{}")
```

Comment:
Yes `_.isEmpty` is perfect :)

Here, I am checking if meta contains something because, in the previous code whatever if we had something we sent it to CloudWatch Log. The "issue" was if we didn't have any meta data we have something like this in our log :
`level - message - {}`
This empty bracket wasn't beautiful, that's why I am checking if the meta data.